provider/packet/userdata.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015 Canonical Ltd.
+// Copyright 2015 Cloudbase Solutions SRL
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package packet
+
+import (
+	"github.com/juju/errors"
+	jujuos "github.com/juju/os"
+
+	"github.com/juju/juju/cloudconfig/cloudinit"
+	"github.com/juju/juju/cloudconfig/providerinit/renderers"
+)
+
+type PacketRenderer struct{}
+
+func (PacketRenderer) Render(cfg cloudinit.CloudConfig, os jujuos.OSType) ([]byte, error) {
+	switch os {
+	case jujuos.Ubuntu, jujuos.CentOS:
```

Comment:
The user data seems ubuntu-specific. Not really sure if you should include CentOS here